### PR TITLE
CNV-12858: Release note SVVP

### PR DESCRIPTION
**Pull request by sabrinajess**
Re: [CNV-12858](https://issues.redhat.com/browse/CNV-12858)
For 4.9 release notes 

Preview: https://deploy-preview-36353--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-9-release-notes

### CODE REVIEW

File: virt/virt-4-9-release-notes.adoc [hunk 1]
```diff
@@ -42,7 +42,14 @@ Red Hat is committed to replacing problematic language in our code, documentatio
...
 
-//CNV-12858 SVVP previous 8914
+//CNV-12858
+* {VirtProductName} is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS__.
+** Intel and AMD CPUs.
+* The Containerized Data Importer (CDI) now uses the {product-title} xref:../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[cluster-wide proxy configuration].
```
Comment (ILpinto) [inline virt/virt-4-9-release-notes.adoc:52]: In the last release note (cnv-4.8) we did not add the CDI version, do we need it?

Comment (sabrinajess) [reply virt/virt-4-9-release-notes.adoc:52]: Whoops, that line is a hitchhiker from another item on the 4.8 RN. Will remove.

**Comment by sabrinajess** [comment]
@ILpinto  Ready for review again.

**Comment by kbidarkar** [comment]
LGTM

**Comment by lijinlijin** [comment]
LGTM

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

**Pull request merged**